drivers/classic/src/get_global_param.c

```diff
@@ -981,23 +1119,25 @@ get_global_param(FILE *gp)
         else if (global_param.endmonth > MONTHS_PER_YEAR) {
             log_err("The specified simulation end month (%hu) < 0.  Make sure "
                     "that the global file defines a positive integer for "
-                    "ENDMONTH.", global_param.endmonth);
+                    "ENDMONTH.",
+                    global_param.endmonth);
         }
         if (global_param.endday == 0) {
             log_err("Simulation end day has not been defined.  Make sure "
                     "that the global file defines ENDDAY.");
         }
-        else if (global_param.endday > lastday[global_param.endmonth]) {
+        else if (global_param.endday > lastday[global_param.endmonth - 1]) {
             log_err("The specified simulation end day (%hu) > the number of "
                     "days in the ENDMONTH (%hu).  Make sure that the global "
                     "file defines a positive integer for ENDDAY.",
```

Comment:
I think it has always been `[1-12]`.  Looking at the [current implementation](https://github.com/UW-Hydro/VIC/blob/ff960db885d8488f9a8faf535ae96e32d48265b4/drivers/classic/src/get_global_param.c), this may have been a existing bug.